src/components/PrintPage.vue

```diff
@@ -5,27 +5,26 @@
 
 <template>
   <div id="print-output">
-
     <transition :duration="500" name="fade">
       <bar-loader :loading="state.loading && state.layers"/>
     </transition>
 
     <template v-if="state.layers">
-        <!-- PRINT as PDF -->
+        <!-- PRINT as PDF or GEOPDF-->
         <iframe
-          v-if  = "'pdf' === state.format"
+          v-if  = "['pdf', 'geopdf'].includes(format)"
           ref   = "out"
           :src = "state.url"
-        ></iframe>
```

Comment:
@volterra79 [iframe](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe) is not a self closing tag

Reply:
@Raruto fix with https://github.com/g3w-suite/g3w-client/pull/623/commits/9cfd839bbbbd902279d7851bfbcd66cc5ca120a5